File: Java/libraries/recognizers-text-number-with-unit/src/main/java/com/microsoft/recognizers/text/numberwithunit/models/CurrencyUnitValue.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.microsoft.recognizers.text.numberwithunit.models;

public class CurrencyUnitValue {

    public final String number;
    public final String unit;
    public final String isoCurrency;

    public CurrencyUnitValue(String number, String unit, String isoCurrency) {
        this.number = number;
        this.unit = unit;
        this.isoCurrency = isoCurrency;
    }
}

File: .NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/CurrencyUnitValue.cs
using System.Collections.Generic;

using Microsoft.Recognizers.Text.NumberWithUnit.Utilities;

namespace Microsoft.Recognizers.Text.NumberWithUnit
{
    public class CurrencyUnitValue : UnitValue
    {
        public string IsoCurrency { get; set; } = string.Empty;
    }
}